tests/Resources/ti.ui.webview.addontest.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+const should = require('./utilities/assertions');
+const utilities = require('./utilities/utilities');
+
+describe('Titanium.UI.WebView', function () {
+	let win;
+	this.slow(3000);
+	this.timeout(30000);
+
+	afterEach(function (done) {
+		if (win) {
+			// If `win` is already closed, we're done.
+			let t = setTimeout(function () {
+				if (win) {
+					win = null;
+					done();
+				}
+			}, 3000);
+
+			win.addEventListener('close', function listener () {
+				clearTimeout(t);
+
+				if (win) {
+					win.removeEventListener('close', listener);
+				}
+				win = null;
+				done();
+			});
+			win.close();
+		} else {
+			win = null;
+			done();
+		}
+	});
+
+	it('progress event', function (finish) {
+		this.slow(5000);
+		this.timeout(10000);
+
+		win = Ti.UI.createWindow();
+		const webView = Ti.UI.createWebView({
+			url: 'https://www.google.com'
+		});
+		const isIOS = utilities.isIOS();
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.webview.addontest.js;
  Line: 52;
-->

- :warning: tests/Resources/ti.ui.webview.addontest.js line 52 – 'isIOS' is assigned a value but never used. (no-unused-vars)